openpathsampling/tests/test_toy_dynamics.py

```diff
@@ -177,57 +175,19 @@ def test_sanity(self):
     def test_snapshot_timestep(self):
         assert_equal(self.sim.snapshot_timestep, 0.02)
 
-    def test_momentum_getter(self):
-        momentum = self.sim.momentum
-        assert_items_equal(momentum.velocities[0],
-                           self.sim.velocities)
-        assert_equal(momentum.kinetic_energy,
-                     self.sim.pes.kinetic_energy(self.sim))
-    
-    def test_momentum_setter(self):
-        raise SkipTest()
-        self.sim.momentum = Momentum(velocities=np.array([[4, 5, 6]]))
-        assert_items_equal(self.sim.velocities, [4, 5])
-
-    def test_configuration_setter(self):
-        raise SkipTest()
-        # This should not work anymore
-        self.sim.configuration = Configuration(
-            coordinates=np.array([[1, 2, 3]])
-        )
-        assert_items_equal(self.sim.positions, [1, 2])
-
-
-    def test_load_configuration(self):
-        configuration = self.sim.configuration
-        assert_items_equal(configuration.coordinates[0],
-                           self.sim.positions)
-        assert_equal(configuration.potential_energy,
-                     self.sim.pes.V(self.sim))
-        assert_equal(configuration.box_vectors, None)
-
     def test_snapshot_get(self):
         snapshot = self.sim.current_snapshot
-        n_spatial=self.sim.n_spatial
-        assert_items_equal(snapshot.momentum.velocities[0],
+        assert_items_equal(snapshot.velocities[0],
                            self.sim.velocities)
-        assert_equal(snapshot.momentum.kinetic_energy,
-                     self.sim.pes.kinetic_energy(self.sim))
-        assert_items_equal(snapshot.configuration.coordinates[0],
+        assert_items_equal(snapshot.coordinates[0],
                            self.sim.positions)
-        assert_equal(snapshot.configuration.potential_energy,
-                     self.sim.pes.V(self.sim))
-        assert_equal(snapshot.configuration.box_vectors, None)
-        
+
     def test_snapshot_set(self):
-        raise SkipTest()
-        snap = Snapshot(coordinates=np.array([[1,2,3]]), 
+        snap = ToySnapshot(coordinates=np.array([[1,2,3]]),
                         velocities=np.array([[4,5,6]]))
-        # note that we truncate the z-direction, regardless of what it is.
-        # This is for a 2D model!
         self.sim.current_snapshot = snap
-        assert_items_equal(self.sim.positions, [1,2])
-        assert_items_equal(self.sim.velocities, [4,5])
+        assert_items_equal(self.sim.positions, [1,2,3])
+        assert_items_equal(self.sim.velocities, [4,5,6])
 
```

Comment:
The idea from the original is that file formats are often explicitly 3D (needed for visualization in VMD, for example). So if I wanted to take a toy system that actually has 2 spatial dimensions and get data in/out from, e.g., XYZ format, I need to remove the z-coordinate at some point. That was what this did.

I frequently did this sort of thing with my 2D "Geisslerene" model, so that's the motivation. Maybe there's a better way to do this, but that was the whole purpose of that specific test.